.changelog/9920.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+connect: The builtin connect proxy no longer advertises support for h2 via ALPN.
```

Comment:
```suggestion
connect: The builtin connect proxy no longer advertises support for h2 via ALPN. [[GH-4466](https://github.com/hashicorp/consul/issues/4466)].
```

Thanks for adding the changelog entry, this looks great just suggesting addition of link back to the issue for context!